.env

```diff
@@ -31,7 +31,7 @@ MAILER_URL=null://localhost
 # Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
 # Configure your db driver and server_version in config/packages/doctrine.yaml
-DATABASE_URL=#mysql://db_user:db_password@127.0.0.1:3306/db_name
```

Comment:
There were some problems at `composer install` when setting this to a non-existing database. Have to check tomorrow what exactly that was.